repos/system_upgrade/el7toel8/actors/setuptargetrepos/actor.py

```diff
@@ -16,10 +21,29 @@ class SetupTargetRepos(Actor):
     """
 
     name = 'setuptargetrepos'
-    consumes = (CustomTargetRepository, RepositoriesSetupTasks, RepositoriesMap, RepositoriesFacts, RepositoriesBlacklisted)
-    produces = (TargetRepositories,)
+    consumes = (CustomTargetRepository,
+                RepositoriesSetupTasks,
+                RepositoriesMap,
+                RepositoriesFacts,
+                RepositoriesBlacklisted,
+                RepositoriesUsage)
+    produces = (TargetRepositories, Report,)
     tags = (IPUWorkflowTag, ChecksPhaseTag)
 
+
+    def report_skipped_repos(self, repos, pkgs):
+        title = 'Some enabled RPM repositories are not compatible with Leapp'
+        summary_data = []
+        summary_data.append('Following repositories are unknown to Leapp:')
+        summary_data.extend(['- ' + r for r in repos])
+        summary_data.append('And the following installed packages from those repositories may not be upgraded:')
+        summary_data.extend(['- ' + p for p in pkgs])
+        summary = '\n'.join(summary_data)
+        reporting.report_generic(title=title, summary=summary, severity='low')
```

Comment:
Add a remediation: "You can file a request to add this repository to the scope of in-place upgrades by filing a support ticket"